src/degree_two_reduction.jl

```diff
@@ -1,13 +1,22 @@
-#NOTE: hardcoded for testing system 
+#NOTE: hardcoded for testing system
 function get_reduction(
-    A::IncidenceMatrix,
+    A::AdjacencyMatrix,
     sys::PSY.System,
     ::Val{NetworkReductionTypes.DEGREE_TWO},
 )
+    irreducible_buses = Set{Int}()
```

Comment:
Note to myself, this selection can consider active or reactive power. so that we can eliminate buses with reactive power support devices.